packages/icon/README.md

```diff
@@ -5,14 +5,9 @@ group: Foundations
 ---
 
 ```jsx live
-<Stack gap={1} flexDirection="row" margin={1}>
-	<AvatarIcon size={3} />
-	<ProgressTodoIcon size={3} />
-	<ProgressDoingIcon size={3} />
-	<ProgressDoneIcon size={3} />
-</Stack>
+<AvatarIcon size="md" weight="regular" />
```

Comment:
I think we should just show off 1 icon here to show the code example is super simple.